pkg/metaserver/agent/metric/metric.go

```diff
@@ -584,9 +614,9 @@ func (m *MalachiteMetricsFetcher) processCgroupPerfData(podUID, containerName st
 	if perf == nil {
 		return
 	}
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCPIContainer, perf.Cpi)
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCyclesContainer, perf.Cycles)
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUInstructionsContainer, perf.Instructions)
+	// m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCPIContainer, perf.Cpi)
```

Comment:
It's not a good idea to use comments to delete code.

Reply:
fixed